createClass.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Create Class</title>
+</head>
+<body>
+	<h1>CREATE COURSE</h1>
+	<form id="xcelform" method="post">
+		Class Name:
+		<input type="text" id="className" required>
+		<br>
+		Stream:
+		<select id="streamSelect" name="streamSelect" form="xcelform">
+			<option value="sample">SAMPLE</option>
+		</select>
+		<br>
+		Upload Xcel File:
+		<br>
+		<input type="file" id="xcelfile" accept=".xcel" required title=".xcel">

```

Comment:
Uuumm..... that is not a valid file extension. You want `.xlsx` and maybe `.xls`. See https://en.wikipedia.org/wiki/List_of_Microsoft_Office_filename_extensions